openverse_catalog/dags/storage/test_image.py

```diff
@@ -140,23 +152,52 @@ def test_create_tsv_row_non_none_if_req_fields(
     assert actual_row is not None
 
 
-@pytest.mark.parametrize(
-    "missing_field",
-    [
-        "foreign_landing_url",
-        "license_",
-        "license_version",
-        "image_url",
-    ],
-)
-def test_create_tsv_row_none_if_missing_required(
+def test_create_tsv_row_none_if_no_foreign_landing_url(
```

Comment:
Looks like this change might have gotten reverted

Reply:
Yes, it did, thank you for spotting it!